tst/TGCellModelGraph.tst

```diff
@@ -355,6 +405,52 @@ gap> CellEdges(tmg);
   [ 8, 9, [ 2, [ 6, 19, 20 ] ], g2*g3^-1 ], [ 12, 4, [ 2, [ 6, 20, 8 ] ], g1^-1*g2 ], 
   [ 9, 11, [ 2, [ 6, 8, 10 ] ], g4^-1*g1^-1 ], [ 4, 15, [ 2, [ 6, 10, 24 ] ], g3*g4^-1 ], 
   [ 11, 6, [ 2, [ 6, 24, 14 ] ], g2^-1*g3 ], [ 15, 13, [ 2, [ 6, 14, 12 ] ], g1*g2^-1 ] ]
+#@else
+gap> tmg := TessellationModelGraph(cg, false : simplify := 5);;
+gap> FlushCaches();
```

Comment:
Why do yo flush the caches here (and below in similar situations)? What happens if the user does not do that? Is it necessary?

Reply:
Please explain this.

Reply:
FlushCaches is used to ensure that the functions SimplifyMethodMessenger@ and rwsFunc@ are tested. More concretely, these functions store internally used objects in cache, through the function MemoizePosIntFunction, a GAP implementation. As such, the use of FlushCaches ensure the actual function is tested and not the MemoizePosIntFunction.

Reply:
Ok, I see, thanks.